### ISSUE TITLE
integrate-detection

### ISSUE THREAD

**Issue by guillaume-byte** [GrayboxTech/weightslab#39]
The current version of the SDK is compatible with detection models, such as YOLO. You can use the Ultralytics version or the one implemented in Torch in baseline_models/pytorch.

The idea is to be able to:
  - Train a detection model (we should be able to play with its architecture now).
  - Visualize data, prediction, and labels

**Comment by guillaume-byte** [comment]
@AlexGrayBox do not hesitate to contribute to the issue with ideas, goals, and everything around the detection integration you have in mind

**Comment by guillaume-byte** [comment]
Pseudo-Detection, i.e., detection and then convert to segmentation maps to be shown as segmentation: https://github.com/GrayboxTech/weightslab/commit/7bbee8f31ce96c707a201e46bfb2772095cb17ea
